pkg/build/rpm_test.go

```diff
@@ -117,9 +121,110 @@ func TestCopyRPMsNoRPMDir(t *testing.T) {
 	require.NoError(t, err)
 	defer os.Remove(builder.eibBuildDir)
 
+	rpmSourceDir := filepath.Join(builder.buildConfig.ImageConfigDir, "rpms")
+	rpmDestDir := builder.combustionDir
+
+	// Test
+	err = builder.copyRPMs(rpmSourceDir, rpmDestDir, nil)
+
+	// Verify
+	require.NoError(t, err)
+}
+
+func TestWriteRPMScript(t *testing.T) {
+	// Setup
+	tmpDir, err := os.MkdirTemp("", "eib-")
+	require.NoError(t, err)
+	defer os.RemoveAll(tmpDir)
+
+	buildConfig := config.BuildConfig{
+		ImageConfigDir: "../config/testdata",
+	}
+	builder := New(nil, &buildConfig)
+	require.NoError(t, builder.prepareBuildDir())
+
+	rpmSourceDir := filepath.Join(builder.buildConfig.ImageConfigDir, "rpms")
+
+	file1Path := filepath.Join(rpmSourceDir, "rpm1.rpm")
+	defer os.Remove(file1Path)
```

Comment:
I believe this should be after the `Create()` call, since if that fails there won't be a file to remove. I'm not sure what will happen if you leave them in this order (`Remove()` may simply exit cleanly since the file isn't there), but from an anal-retentive point of view, defering the remove before the file is created feels weird.